.github/workflows/prerelease.yml

```diff
@@ -5,16 +5,20 @@ on:
   schedule:
     - cron: '0 0 * * 3'
   workflow_dispatch:
-    inputs:
```

Comment:
Out of curiosity, what resolved the intermittent test failures?

Reply:
Intermittent failures haven't been resolved. There are a few that happen somewhat rarely.

However, all of the persistent failures have been resolved. The only one I resolved personally was the Azure Peering test, which was failing due to bad credentials in the CI environment

Reply:
Oh, there were also some failures caused by HVN resources that were not cleaned up in very old test runs. Removing them from the HCP UI solved that issue.

Reply:
Thanks for the info! 👍🏻